.circleci/config.yml

```
@@ -2,7 +2,7 @@
 # syntax, such as anchors, will be fixed automatically.
 version: 2.1
 orbs:
-  shared: getoutreach/shared@2.24.0
+  shared: getoutreach/shared@dev:first
```

suggestion: probably avoid using the dev version since it expires after 30 days since the last commit.

```suggestion
  shared: getoutreach/shared@2.25.1
```

---

Closing as this includes the Golang 1.22 changes, release a new PR for stencil-golang